services/ec2/src/main/java/org/finra/gatekeeper/services/accessrequest/model/activerequest/ActiveAccessConsolidated.java

```diff
@@ -0,0 +1,43 @@
+package org.finra.gatekeeper.services.accessrequest.model.activerequest;
+
+import java.util.List;
+
+public class ActiveAccessConsolidated {
+
+    private List<ActiveAccessRequest> linux;
+    private List<ActiveAccessRequest> windows;
+
+    public ActiveAccessConsolidated() {
```

Comment:
You should initialize the arraylists as empty in the constructor, I see you creating objects to populate these values  every time you instantiate this object